src/components/AnimatedWallet/AnimatedWallet.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { View } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+import { Colors } from '../../theme';
+import { useAnimatedWallet } from './hook';
+import styles from './styles';
+import type { AnimatedWalletStyle } from './types';
+
+const AnimatedWallet = ({
+  size = 250,
+  primaryColor = Colors.red,
+  secondaryColor = Colors.orange,
+  numberOfCards = 1,
+}: Partial<AnimatedWalletStyle>) => {
+  const {
+    AnimatedGradientView,
+    walletPinAnimation,
+    card1Animation,
+    card2Animation,
+    card3Animation,
+    lighterPrimaryColor,
+    lighterSecondaryColor,
+    gradientPosition,
+  } = useAnimatedWallet({ size, primaryColor, secondaryColor });
+
+  const style = styles({
+    size,
+  });
+
+  return (
+    <View style={style.container}>
+      <AnimatedGradientView
+        colors={[primaryColor, lighterPrimaryColor]}
+        style={style.wallet}
+        {...gradientPosition}
+      >
+        <LinearGradient
+          colors={[secondaryColor, lighterSecondaryColor]}
+          style={style.walletPart}
+          {...gradientPosition}
+        />
+        <AnimatedGradientView
+          colors={[lighterSecondaryColor, secondaryColor]}
+          style={[style.walletButton, walletPinAnimation]}
+          {...gradientPosition}
+        >
+          <AnimatedGradientView
+            colors={[lighterPrimaryColor, primaryColor]}
+            style={[style.circle]}
+            {...gradientPosition}
+          />
+        </AnimatedGradientView>
+      </AnimatedGradientView>
+      {(numberOfCards === 1 || numberOfCards === 2 || numberOfCards === 3) && (
+        <AnimatedGradientView
+          colors={[primaryColor, lighterPrimaryColor, primaryColor]}
+          style={[{ ...style.card }, style.card1, card1Animation]}
```

Comment:
update the name of style and animation variable

Reply:
Resolved